pages/Documents/RichMessaging/AppleMessagesForBusinessTemplates/form-message-template.md

```diff
@@ -0,0 +1,583 @@
+      - **emailAddress**: Specifies a keyboard optimized for entering email addresses. This keyboard type prominently features the at (@), period (.), and space characters.
+      - **decimalPad**: Specifies a keyboard with numbers and a decimal point.
+      - **UIKeyboardTypeTwitter**: Specifies a keyboard optimized for Twitter text entry, with easy access to the at (@) and hash tag (#) characters.
+      - **webSearch**: Specifies a keyboard optimized for web search terms and URL entry. This keyboard type prominently features the space and period (.) characters.
+      - **textContentType**: A string value representing the keyboard and system information about the expected semantic meaning for the content that users enter. Supported values are name, namePrefix, givenName, middleName, familyName, nameSuffix, nickname, jobTitle, organizationName, location, fullStreetAddress, streetAddressLine1, streetAddressLine2, addressCity, addressState, addressCityAndState, sublocality, countryName, postalCode, telephoneNumber, emailAddress, URL, creditCardNumber, username, password, newPassword, and oneTimeCode. </td></tr>
+</tbody>
+  </table>
+
+  For all possible options for page configuration check Apple Documentation [here](https://register.apple.com/resources/messages/msp-rest-api/type-interactive#form-message).
+
+#### Example Page Template
+
+```json
+{
+    "pageIdentifier": "1",
+    "type": "input",
+    "nextPageIdentifier": "2",
+    "title": "Your email address",
+    "subtitle": "Enter your email address",
+    "options": {
+        "required": true,
+        "inputType": "singleline",
+        "maximumCharacterCount": 250,
+        "keyboardType": "emailAddress",
+        "textContentType": "emailAddress"
+    }
+}
+```
+
+### Example Body Template
+The structured content body will allow you to define spalsh screen of the Business Form. 
+
+Following Properties must be included in Body template
+* type
+* tag
+* elements
+
+Every splash screen contains title, subtitle and button, these must be included in elements section.
+
+```json
+{
+    "type": "vertical",
+    "tag": "form",
+    "elements": [
+        {
+            "type": "text",
+            "text": "Shipping Information",
+            "tag": "title",
+            "style": {
+                "bold": true,
+                "size": "large"
+            }
+        },
+        {
+            "type": "text",
+            "tag": "subtitle",
+            "text": "For Courier Shipping, all information must be provided."
+        },
+        {
+            "type": "button",
+            "title": "Continue"
+        }
+    ]
+}
+```
+
+### Example Metadata Template
+
+The structured content metadata will allow you to define pages and visual appreance of the form.
+The Metadata template consists of two sections
+* BusinessFormEvent
+* BusinessChatMessage
+
+**Metadata Properties (BusinessFormEvent)**
+
+<table>
```

Comment:
Please switch to Markdown, too. (All tables.)

Reply:
Implemented requested change